apps/balancer-tools/src/app/apr/(components)/OverviewCards.tsx

```diff
@@ -0,0 +1,20 @@
+import { KPI } from "#/components/KPI";
+
+interface CardDetail {
+  title: string;
+  content: string;
+}
+
+function OverviewCards({ cardsDetails }: { cardsDetails: CardDetail[] }) {
+  return (
+    <div className="flex justify-between border border-blue6 bg-blue3 rounded p-4">
+      <div className="flex flex-row flex-wrap xl:flex-nowrap gap-6 justify-evenly w-full">
```

Comment:
cards should be all the same width, and stack 2/2 on smaller breakpoints.https://www.radix-ui.com/ has a nice example showing that -- I also shared a screenshot in the #bal-bribes-x channel.